cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -524,7 +524,7 @@ ELSEIF (TPL_ENABLE_Netcdf )
       SET(PNETCDF_ROOT "$ENV{PNETCDF_ROOT}")
     ENDIF()
     ASSERT_DEFINED(ENV{ZLIB_ROOT})
-    ATDM_SET_CACHE(TPL_Netcdf_LIBRARIES "$ENV{NETCDF_ROOT}/lib/libnetcdf.${ATDM_TPL_LIB_EXT};${PNETCDF_ROOT}/lib/libpnetcdf.a;${HDF5_ROOT}/lib/libhdf5_hl.${ATDM_TPL_LIB_EXT};${HDF5_ROOT}/lib/libhdf5.${ATDM_TPL_LIB_EXT};$ENV{ZLIB_ROOT}/lib/libz.${ATDM_TPL_LIB_EXT};-ldl;-lcurl" CACHE FILEPATH)
+    ATDM_SET_CACHE(TPL_Netcdf_LIBRARIES "$ENV{NETCDF_ROOT}/lib/libnetcdf.${ATDM_TPL_LIB_EXT};${PNETCDF_ROOT}/lib/libpnetcdf.a;${HDF5_ROOT}/lib/libhdf5_hl.${ATDM_TPL_LIB_EXT};${HDF5_ROOT}/lib/libhdf5.${ATDM_TPL_LIB_EXT};$ENV{ZLIB_ROOT}/lib/libz.${ATDM_TPL_LIB_EXT};-ldl" CACHE FILEPATH)
```

Comment:
Note that if `ATDM_USE_SPARC_TPL_FIND_SETTINGS` is true, then this statement is never even called!  That is the case for the ATDM Trilinos 'ats2' configuration so this change does not impact the ATDM Trilinos 'ats2' configuration at all.

Reply:
Ah, I think EMPIRE explicitly unsets `ATDM_USE_SPARC_TPL_FIND_SETTINGS` after sourcing the ATDM env script.

Reply:
@dc-snl, changing the value of `ATDM_USE_SPARC_TPL_FIND_SETTINGS` and tweaking other options makes this no longer an ATDM Trilinos 'ats2' configuration.  What that means is that the automated builds of the 'ats2' configuration being posted daily to [the Trilinos CDash site](https://testing.sandia.gov/cdash/index.php?project=Trilinos&filtercount=1&showfilters=1&field1=buildname&compare1=65&value1=Trilinos-atdm-ats2_) no longer are testing the configurations that are being used by customer APPs.  If each APP team is going to have its own custom Trilinos configuration tweaks, then we are just going back to the days where each APP maintains their own Trilinos configuration.  If that is the case, then these automated builds should be taken down and the APP teams should go back to maintaining their own configurations (which is where this seems to be going anyway).